app/views/spotlight/translations/show.yaml.yamlbuilder

```diff
@@ -1,20 +1,20 @@
-json.set!(I18n.locale) do
+json.set!(@language) do
   json.set!(current_exhibit.slug) do
-    json.title(current_exhibit[:title])
-    json.subtitle(current_exhibit[:subtitle])
-    json.description(current_exhibit[:description])
+    json.title(current_exhibit[:title]) if Translation.find_by(exhibit: current_exhibit, key: "#{current_exhibit.slug}.title", locale: @language).present?
+    json.subtitle(current_exhibit[:subtitle]) if Translation.find_by(exhibit: current_exhibit, key: "#{current_exhibit.slug}.subtitle", locale: @language).present?
+    json.description(current_exhibit[:description]) if Translation.find_by(exhibit: current_exhibit, key: "#{current_exhibit.slug}.description", locale: @language).present?
   end
 
   json.spotlight do
     json.curation do
       json.nav do
-        json.home(t(:'spotlight.curation.nav.home', locale: I18n.default_locale))
+        json.home(t(:'spotlight.curation.nav.home', locale: @language)) if try(t(:'spotlight.curation.nav.home', locale: @language))
```

Comment:
This is also admittedly weird. I try tacking a `.present?` on the key lookup I get a translation missing error interpolated instead.
```
  spotlight:
    catalog:
      breadcrumb:
        index: '<span class="translation_missing" title="translation missing: es.spotlight.catalog.breadcrumb.index,
          locale: es">Index</span>'
```